games/nasb2/base_files/config.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "Nickelodeon All-Star Brawl 2",
+  "challonge_game_id": null,
+  "smashgg_game_id": 39281,
+  "character_to_codename": {
+    "SpongeBob": {
+      "codename": "SpongeBob"
+    },
+    "Patrick": {
+      "codename": "Patrick"
+    },
+    "Squidward": {
+      "codename": "Squidward"
+    },
+    "Mecha Plankton": {
+      "codename": "MechaPlankton"
+    },
+    "El Tigre": {
+      "codename": "ElTigre"
+    },
+    "Rocko": {
+      "codename": "Rocko"
+    },
+    "Jimmy Neutron": {
+      "codename": "Jimmy"
+    },
+    "Lucy Loud": {
+      "codename": "Lucy"
+    },
+    "Angry Beavers": {
+      "codename": "Dagget"
+    },
+    "Garfield": {
+      "codename": "Garfield"
+    },
+    "Aang": {
+      "codename": "Aang"
+    },
+    "Korra": {
+      "codename": "Korra"
+    },
+    "Azula": {
+      "codename": "Azula"
+    },
+    "Raphael": {
+      "codename": "Raphael"
+    },
+    "Donatello": {
+      "codename": "Donatello"
+    },
+    "April": {
+      "codename": "April"
+    },
+    "Danny Phantom": {
+      "codename": "Danny"
+    },
+    "Ember": {
+      "codename": "Ember"
+    },
+    "Grandma Gertie": {
+      "codename": "GrandmaGertie"
+    },
+    "Gerald": {
+      "codename": "Gerald"
+    },
+    "Nigel": {
+      "codename": "Nigel"
+    },
+    "Zim": {
+      "codename": "Zim"
+    },
+    "Jenny Wakeman": {
+      "codename": "Jenny"
+    },
+    "Reptar": {
+      "codename": "Reptar"
+    },
+    "Ren & Stimpy": {
+      "codename": "RenStimpy"
+    }
+  },
+
+  "stage_to_codename": {},
```

Comment:
I found a new bug: The stages aren’t defined in `base_files/config.json` and therefore don’t appear in the program

![Screenshot_1449](https://github.com/joaorb64/StreamHelperAssets/assets/1964201/a220f775-1ac0-40e0-9d4f-f8cb4bd3b1b3)